### ISSUE TITLE
Allow fallback to hosted js files without cdn

### ISSUE THREAD

**Issue by ofbeaton** [graphql/express-graphql#112]
Tried to use graphiql provided by this package on an intranet without an internet connection, horrible experience, everything is loaded from cdn.

Similar issues in other flavours of graphiql, seems like I'm not the only one having this issue.
exponentjs/koa-graphiql#1

This was already fixed in the `example/` for graphiql itself: graphql/graphiql#135

Could we please have a fallback where if the cdn fails to load the files it loads a local copy?

**Comment by leebyron** [comment]
Good idea. Should be able to pass something like "local" or "cdn" to the graphiql option

This would be a good pick up task for anyone interested in contributing.

**Comment by ofbeaton** [comment]
or perhaps something like bellow? Though the advantage of the local flag is that it wouldn't even bother trying to hit the external network.

```
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="js/vendor/jquery.min.js') }}"><\/script>')</script>
```

**Comment by ilia-luk** [comment]
+1

**Comment by KyleAMathews** [comment]
Or just make GraphiQL a react app and compile it out to a bundle.js.

**Comment by dgrcode** [comment]
I'm happy to work on this if nobody is already doing it

**Comment by tanaypratap** [comment]
great. I will be working on this and putting up a draft peer soon. Thanks for the quick reply @dgrcode

**Comment by IvanGoncharov** [comment]
CDN usage was completely removed in #524
